docs/index.md

```diff
@@ -24,7 +24,7 @@ Civil Aviation Authorities worldwide have started rule making for unmanned aviat
 | Siddharth Ravikumar  | [<i class="fa fa-envelope-o"></i>](mailto:ravikumar.siddharth@gmail.com)                     |      | iSpirt                           | 🇨🇦     |
 | Amit Garg            | [<i class="fa fa-envelope-o"></i>](mailto:studies.amit@gmail.com)                            |      | iSpirt                           | 🇮🇳     |
 | George Thomas        | [<i class="fa fa-envelope-o"></i>](mailto:georj13@gmail.com)                                 |      | iSpirt                           | 🇮🇳     |
-| Hrishikesh Ballal    | <a href="https://about.openskies.sh/#contact" target="_blank"><i class="fa fa-link"></i></a> |      | Openskies Aerial Technology Ltd. | 🇮🇪     |
-| Sayandeep Purkayasth | [<i class="fa fa-envelope-o"></i>](mailto:sayandeep@deepcyan.ai)                             |      | Deepcyan Software                | 🇮🇳     |
+| Hrishikesh Ballal    | <a href="https://about.openskies.sh/#contact" target="_blank"><i class="fa fa-link"></i></a> |      | iSpirt                           | 🇮🇪     |
+| Sayandeep Purkayasth | [<i class="fa fa-envelope-o"></i>](mailto:sayandeep@deepcyan.ai)                             |      | iSpirt                           | 🇮🇳     |
```

Comment:
I'm comfortable having both: iSpirt & Deepcyan.

Reply:
In fact, we should advertise that we have varied membership, so the whole workstream doesn't get categorised as biased or opinionated.